src/styles/createTheme.tsx

```diff
@@ -8,6 +8,15 @@ const green = '#1D878C'
 
 const theme: Record<string, any> = createTheme({
   spacing: 5,
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 600,
+      md: 900,
```

Comment:
Do we need to specify values `xs, sm and md`? We don't change these values. These are default breakpoints.

Reply:
Typescript provides an error if I use remove it there.
<img width="574" alt="image" src="https://github.com/WeFriiends/WeFriiendsWeb/assets/324031/1f0a31ce-395f-49f9-accd-4ec8e7d8a025">

Reply:
Ok, understood